app/views/admin/users/index.html.haml

```diff
@@ -10,7 +10,7 @@
     %table.table.table-striped.table-bordered.table-hover.datatable
       %thead
         %th ID
-        %th State
+        %th Confirmation State
```

Comment:
Not out of the box. But Confirmation State 'Yes' also doesn't make a lot of sense.  Perhaps the %th should be `Confirmed?` And then Yes/No will make more sense.